common/src/warp_runner/manager/commands/multipass_commands.rs

```diff
@@ -459,6 +400,26 @@ pub async fn handle_multipass_cmd(cmd: MultiPassCmd, warp: &mut super::super::Wa
     }
 }
 
+async fn update_identity_with_correct_values(
```

Comment:
To be more consistent with the actual task since this isnt about updating with "correct values" but more just mapping data out to the `Identity` wrapper.

```suggestion
async fn update_identity(
```